projects/docs-tools/src/modules/demo-page/demo-page.component.html

```diff
@@ -20,6 +20,20 @@
           <ng-container [ngTemplateOutlet]="content"></ng-container>
         </sky-tab>
 
+        <sky-tab tabHeading="Testing" *ngIf="enableTestingTab">
+          <sky-docs-demo-page-section heading="Test harnesses">
+            <p>
+              SKY UX test harnesses are built upon Angular CDK component
+              harnesses. For more information see the
+              <a href="https://material.angular.io/cdk/test-harnesses/overview"
+                >Angular CDK component harness documentation.</a
```

Comment:
The period should exist outside of the anchor tag.

Reply:
Done